src/report_workflow/report_args.py

```diff
@@ -33,6 +35,8 @@ def __init__(self) -> None:
         parser.add_argument("--output-path", type=str, help="Specify the path location for the test-report manifest.")
         parser.add_argument("--test-run-id", type=int, help="The unique execution id for the test")
         parser.add_argument("--component", type=str, dest="components", nargs='*', help="Test a specific component or components instead of the entire distribution.")
+        parser.add_argument("--schema-version", type=str, default="1.1", help="The schema version for the test-report manifest.")
+        parser.add_argument("--release-candidate", type=str, default="0", help="The release candidate (rc) information of the artifacts, added in schema-version='1.1'.")
```

Comment:
Why do we need this? I assume this is the RC number which should be same as the ID we fetched from the bundle manifest right?

Reply:
RC number is not build id they are different.
On Jenkins distribution build release manager manually enter the RC number of the release candidate.

Reply:
Oh I see that was recently introduced into the build job. https://github.com/opensearch-project/opensearch-build/pull/4876
Thanks.